### ISSUE TITLE
Using text review, when navigating by word, unexpected blank item encountered at the end

### ISSUE THREAD

**Issue by nidza07** [nvaccess/nvda#18912]
### Steps to reproduce:
1. Launch the latest NVDA alpha.
2. Using text review commands (NumPad 4/NumPad6 on desktop layout), try navigating word by word across a list item.

### Actual behavior:
When you reach the last word and press NumPad6 to move to the next word, you will hear "Blank". 
### Expected behavior:
NVDA should announce "Bottom" and repeat the last word again.
### NVDA logs, crash dumps and other attachments:

[NVDA broken word navigation.txt](https://github.com/user-attachments/files/22317637/NVDA.broken.word.navigation.txt)

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
alpha-52562,3d74061b (2026.1.0.52562)
#### Windows version:
Windows 11 24H2 (AMD64) build 26100.6584
#### Name and version of other software in use when reproducing the issue:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Works on stable
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by LeonarddeR** [comment]
This might be caused by https://github.com/nvaccess/nvda/pull/18744 or https://github.com/nvaccess/nvda/pull/18348

**Comment by seanbudd** [comment]
Is this still an issue in alpha builds after #19204 was merged?

**Comment by makhlwf** [comment]
> Is this still an issue in alpha builds after #19204 was merged?



Yes it is

**Comment by SaschaCowley** [comment]
@LeonarddeR I've just bisected and it looks like this is caused by #18744. Are you able to look into it?

**Comment by LeonarddeR** [comment]
I can definitely look into this, though I can’t give an ETA. May be next 
week.

&#8203;
